sources/stylesheets/_fonts.scss

```diff
@@ -0,0 +1,530 @@
+@font-face {
```

Comment:
Indentation in this file should be fixed: everything below this line should be indented two spaces to the left.

This file should also have an empty line at the end of the file (so GitHub wouldn't be mad at us).